src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -69,7 +70,7 @@ export class ApiKeyAuthenticator {
 
     return tryAuthenticate()
       .then((response: IResponse) => {
-        const body = response.body;
+        const body = response.body as IHash;
```

Comment:
no difference. in pure js it will be plain object, it just a 'syntax sugar'.
IHash added just for do not have compiler error.
i created it as interface declaration just because cast 'as {[property: string]: any}' is too long and it's used in a few places